django_etebase/app.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "etebase_server.settings")
+application = get_wsgi_application()
+
+from django.contrib.auth import get_user_model
+from fastapi import FastAPI, Depends
+from msgpack_asgi import MessagePackMiddleware
+from django_etebase.authentication import get_authenticated_user
+
+
+User = get_user_model()
+app = FastAPI()
+app.add_middleware(MessagePackMiddleware)
+
+
+@app.get("/v1/test")
+async def test_token(user: User = Depends(get_authenticated_user)):
+    print(user)
+
+
+if __name__ == "__main__":
```

Comment:
Is this a convention with fastAPI? I thought you are just expected to run `uvicorn` yourself like so: `uvicorn django_etebase:app --reload`.
See: https://fastapi.tiangolo.com/#run-it

Reply:
its just for experimenting and easier debugging with pycharm , it shouldn't be there eventually

Reply:
Ahh cool, don't mind me then. :)

Reply:
> For now at least, we should keep the Django ORM, though I'm happy with moving to a more async-friendly ORM if we can easily implement a migration from the current database schema and if the new ORM is well known, well used, and well tested.
>
[https://fastapi.tiangolo.com/advanced/async-sql-databases/](url)
SQLAlchemy looks like a good choice
might be easier to change both together (not sure im right about that)

Reply:
I don't think making two such big changes at once is the right idea. I'd much rather only implement the API, see it actually works, better, we like it. Test it a bit, and only then commit to moving to it with changing the persistence layer as well.
Also, changing such big things at the same time is a recipe for trouble. It essentially amounts to a rewrite, and we will also have to re-implement the django admin and many other things. Not worth it at this point. :)